recipes/diffpy-utils/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "diffpy.utils" %}
+{% set version = "1.2.2" %}
+{% set sha256 = "9cc409762aa40d334bfe49cf6ac5f46a20e718818d1dce41c5972e04c3945abc" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+  skip: True  # [not py27]
+
+requirements:
+  build:
+    - python
+    - setuptools
+
+  run:
+    - python
+    - setuptools
+    - numpy >=1.3
+
+test:
+  imports:
+    - diffpy
+    - diffpy.utils
+    - diffpy.utils.parsers
+    - diffpy.utils.tests
+
+about:
+  home: https://github.com/diffpy/diffpy.utils/
+  summary: Shared utilities for diffpy packages.
+  license: Modified BSD
+  license_family: BSD
+  license_file: LICENSE.txt
+  description: |
+    The diffpy.utils package provides functions for extracting array
+    data from variously formatted text files and wx GUI utilities used by the
+    PDFgui program. The package also includes an interpolation function based on
+    the Whittaker-Shannon formula that can be used to resample a PDF or other
+    profile function over a new grid.
+  doc_url: http://www.diffpy.org/diffpy.utils/
+  dev_url: https://github.com/diffpy/diffpy.utils
+
+extra:
+  recipe-maintainers:
+    - CJ-Wright
+    - pavoljuhas
```

Comment:
@pavoljuhas Are you ok being added as a co-maintainer fo the `diffpy.utils` recipe on conda-forge?